test/test_mps.py

```diff
@@ -9234,6 +9234,22 @@ def test_sdpa_enable_gqa(self, dtype, is_causal):
             )
         self._compare_tensors(y.cpu(), y_ref)
 
+    def test_sdpa_fp32_no_memory_leak(self):
```

Comment:
I think you'll want to decorate this one with `@serialTest` so that pytest will not try to run it in parallel with something else